README.md

```diff
@@ -1,16 +1,24 @@
 # Library Developers
 
 ## Overview
-Use `browzine-summon-adapter` to enhance Summon search results with BrowZine data; Adds links to Journal and Article content in BrowZine; Uses BrowZine Journal covers. The BrowZine Summon adapter supports IE11+ and evergreen browsers (Chrome, Firefox, Safari, Microsoft Edge).
+Use `browzine-summon-adapter` to enhance Summon or 360 Core search results with BrowZine data; Adds links to Journal and Article content in BrowZine; Uses BrowZine Journal covers. The BrowZine Summon adapter supports IE11+ and evergreen browsers (Chrome, Firefox, Safari, Microsoft Edge).
 
+_Summon Search Results_
+
+![Summon Journal Search Results](https://i.imgur.com/B34LEec.png "Summon Journal Search Results")
+
+_360 Core Search Results_
```

Comment:
Nice to see it labeled this way - we've called it SerSol a lot around here, but the labeling on [the official website](http://www.proquest.com/products-services/360-Core.html) is just 360 Core.

Reply:
Yep! If only, we could name a JavaScript object starting w/ numbers. 😂